src/internal/utils/network.go

```diff
@@ -41,6 +45,15 @@ func DownloadToFile(url string, target string) {
 	}
 	defer destinationFile.Close()
 
+	// If the url start with the sget protocol use that, otherwise do a typical GET call
+	if url[:len(SGETProtocol)] == SGETProtocol {
```

Comment:
Interesting way to match the string, [strings.HasPrefix](https://pkg.go.dev/strings#HasPrefix) might be a bit more readable though.